src/task3/task3.js

```diff
@@ -0,0 +1,14 @@
+function isEven(n)
+    {
```

Comment:
Function writing pattern please

Reply:
This function should be much shorter.
Recursion is a key and you use it well !
Best to write in 7 ish lines

Reply:
Oh, ur patience's gonna be rewarded